packages/backend-tasks/src/tasks/TaskWorker.ts

```diff
@@ -292,14 +292,23 @@ export class TaskWorker {
           seconds: dt,
         })}`,
       );
-      nextRun = this.knex.client.config.client.includes('sqlite3')
-        ? this.knex.raw(
-            `max(datetime(next_run_start_at, ?), datetime('now'))`,
-            [`+${dt} seconds`],
-          )
-        : this.knex.raw(
-            `greatest(next_run_start_at + interval '${dt} seconds', now())`,
-          );
+
+      nextRun = this.knex.raw(
+        `greatest(next_run_start_at + interval '${dt} seconds', now())`,
+      );
+
+      if (this.knex.client.config.client.includes('sqlite3')) {
+        nextRun = this.knex.raw(
+          `max(datetime(next_run_start_at, ?), datetime('now'))`,
+          [`+${dt} seconds`],
+        );
+      }
+
+      if (this.knex.client.config.client.includes('mysql')) {
+        nextRun = this.knex.raw(
+          `greatest(next_run_start_at + interval ${dt} second, now())`,
+        );
+      }
```

Comment:
should change this into `if (sqlite) {...} else if (mysql) {...} else {...}`, so we don't needlessly construct a fragment on like 296 that then gets thrown away.